web/app.js

```diff
@@ -1457,6 +1460,9 @@ function webViewerInitialized() {
     if ('useonlycsszoom' in hashParams) {
       PDFJS.useOnlyCssZoom = (hashParams['useonlycsszoom'] === 'true');
     }
+    if ('printautorotate' in hashParams) {
```

Comment:
How would you test the parameter in the web viewer without the hash param? I only added it in order to test it from the test server.